client/src/redux/actions/credentialsActions.js

```diff
@@ -15,10 +16,15 @@ const getCredential = id => dispatch => {
   });
 };
 
-const getCredentials = (query = {}) => dispatch => {
+const getCredentials = (query = {}) => (dispatch, getState) => {
```

Comment:
It might be better to have a separate action for this `getAllCredentials` that could then dispatch a different type which wouldn't be picked up by the credentialsReducer and we wouldn't need to track wizard state here.

Reply:
Coincidentally, we did give a separate action a go as one of the first attempts to avoid having to pass the additional state with a limit on it, feels hacky along with the Reducer condition. We can take this approach, but a compare/contrast from what we saw...

The simple approach we add another action, a single constant, avoid creating a separate Action.js file. From there 2 possibilities pop up...

Either we place a new case in the Credentials reducer, or we place it in the Wizard reducer. 

1. Adding the case to the Credentials reducer means we can cause the Credentials view to refresh when it comes time to updating it. This is nice because...

2. Adding the case to the Wizard reducer means we have to specifically place logic just for the scenario of closing the Wizard in the Credentials view during the initial Empty State. There's a slim chance that could happen since we've placed all the necessarily buttons in there... jokingly we could remove the Add Source button from the Credentials view empty state and that would resolve the issue too =)

The part we tried was the second approach, and that didn't appear necessarily cleaner or an improvement on readability, just shifted the logic around. In working through the first approach, we may be able to eliminate a few bits (like the condition check in the Reducer) and make it more readable, slightly. We're game for that, we'll give it go, readability for the win!